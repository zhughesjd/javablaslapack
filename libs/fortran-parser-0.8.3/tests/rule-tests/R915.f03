

! Testing input item. Should be simple.
read *, truth, beauty
read *, or, ugly, lies 
end
