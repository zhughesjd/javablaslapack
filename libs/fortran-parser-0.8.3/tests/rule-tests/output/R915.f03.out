1101:begin
915:list-begin
613 truth false false
612 1
603 false
601
613 beauty false false
612 1
603 false
601
915:list 2
915:list-begin
613 or false false
612 1
603 false
601
613 ugly false false
612 1
603 false
601
613 lies false false
612 1
603 false
601
915:list 3
1103
1101 false true false
